src/UIState/NumberCollectorState.h

```diff
@@ -84,4 +87,13 @@ class TestIntNumCollectorState : public TestNumCollectorState {
   }
 };
 
+class TestNoCurrentNumCollectorState : public TestNumCollectorState {
```

Comment:
When I read the name of this class I thought that there was no number collector; perhaps the class name could be `TestNumCollectorStateWithNoCurrent` (but it didn't take much to figure out what is intended).